toolkit/data-sources/cli/src/main.rs

```diff
@@ -66,13 +70,19 @@ macro_rules! follower_commands {
 
 follower_commands! {
 	block {
+		/// Returns the latest block recorded by DB-Sync
 		async fn get_latest_block_info();
-		async fn get_latest_stable_block_for(1 reference_timestamp: u64);
-		async fn get_stable_block_for(1 hash: McBlockHash, 2 reference_timestamp: u64);
+		/// Returns data of block that was the latest stable block at given timestamp
+		async fn get_latest_stable_block_for(1 reference_timestamp_millis: u64);
```

Comment:
Added `_millis` postfix here, because it was not obvious and we don't have mechanism for params comments/help.